src/test/java/POM/LoginPage.java

```diff
@@ -0,0 +1,40 @@
+package POM;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class LoginPage extends BasePage {
+    By emailFieldLocator = By.xpath("//input[@type='email']");
+    By passwordFieldLocator = By.xpath("//input[@type='password']");
+    By submitButtonLocator = By.xpath("//button[@type='submit']");
```

Comment:
**I recommend:  make all the locators private**, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

    ```
    private By emailFieldLocator = By.xpath("//input[@type='email']");
    private By passwordFieldLocator = By.xpath("//input[@type='password']");
    private By submitButtonLocator = By.xpath("//button[@type='submit']");
    ```